### PR TITLE
tut 1.0.21 (new formula)

### PR DESCRIPTION
**Pull request by elijahmanor**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/tut.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  desc "TUI for Mastodon with vim inspired keys"
+  homepage "https://tut.anv.nu"
+  url "https://github.com/RasmusLindroth/tut/archive/refs/tags/1.0.21.tar.gz"
+  sha256 "64139c3e8f5873d78e0044f158bf9070b18e0a96cb7ae48a350d75476d9073e9"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"tut"
```
Comment (chenrui333) [inline Formula/tut.rb:11]:
```suggestion
    system "go", "build", *std_go_args
```

File: Formula/tut.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/tut --version")
+    assert_match "TUI for Mastodon", shell_output("#{bin}/tut --help")
```
Comment (chenrui333) [inline Formula/tut.rb:16]: we might need a better test for this. You can find similar UI tests in the repo.

File: Formula/tut.rb (hunk 1 above)
Comment (elijahmanor) [reply Formula/tut.rb]: Done

File: Formula/tut.rb (hunk 2 above)
Comment (elijahmanor) [reply Formula/tut.rb:16]: Thanks for the feedback. I went ahead and tested the `tut --new-user` command that prompts for the mastodon instance. From there you have to authenticate with a browser and that flow is fairly complicated, but this proves that the auth flow starts to work.

Comment (chenrui333) [reply Formula/tut.rb]: That is good! 👍

**Review by chenrui333 — APPROVED** [review]

**Comment by p-linnane** [comment]
Thank you @elijahmanor!

**Pull request closed without merging**